lua/wikis/commons/Widget/CharacterTable/Entry.lua

```diff
@@ -0,0 +1,11 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Widget/CharacterTable/Entry
```

Comment:
following the structure of other modules imho we should name it `Module:Widget/CharacterTable/Entry/Custom`
would also have the advantage of it being in the same folder as the base one :)
(imho wait for rath's opinion though)